us/en/skymiles/blocks/find-useful/find-useful.js

```diff
@@ -0,0 +1,90 @@
+function showAcknowledgeText() {
+  // Replace question with acknowledge text
+  const acknowledgeText = document.getElementById('acknowledge-text');
+  const questionContainer = document.querySelector('.question-container');
+  questionContainer.setAttribute('hidden', 'true');
```

Comment:
You could use `aria-hidden` attribute instead so we have an accessible property here